embedded-graphics/src/geometry/size.rs

```diff
@@ -180,6 +180,11 @@ impl Size {
     pub(crate) fn center_offset(self) -> Self {
         self.saturating_sub(Size::new(1, 1)) / 2
     }
+
+    /// Returns the componentwise minimum of two `Size`s
+    pub(crate) fn component_min(self, other: Self) -> Self {
```

Comment:
Should I add `Point::component_min`, as well as `component_max` for both `Point` and `Size`?

I could also make this `pub`. Thoughts?

Reply:
> Should I add `Point::component_min`, as well as `component_max` for both `Point` and `Size`?

IMO that's a good idea, I could have used these methods before.

> I could also make this `pub`. Thoughts?

I think I would make them `pub`, but I would like to another look at the `Point` and `Size` API before making a final decision. Can you leave them `pub(crate)` for now and open an issue to discuss this change? There were some other possible additions to the API I considered to make some time ago but can't remember at the moment.

Reply:
I've added all the methods and changed them to `pub(crate)` and ticket opened at #338 .